data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/CloudEventDeserializerTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
```

Comment:
Don't want to complain too much about the years here, but can you have a look all the files that are added in this PR?

Reply:
Unfortunately, the plugin that we use to verify the licence header expect a fixed date for all files, so we have set it to 2018.